src/MercadoPago/Entity.php

```diff
@@ -100,7 +100,35 @@ public static function read($params = [])
     /**
      * @return mixed
      */
-    public static function search($filters)
+    public static function all($params = [])
+    { 
+        
+        $class = get_called_class();
+        $entity = new $class();
+        $entities =  array();
```

Comment:
use short syntax for array 

```suggestion
        $entities =  [];
```